src/basic_recipes/poly.jl

```diff
@@ -194,7 +195,9 @@ function plot!(plot::Poly{<: Tuple{<: Union{Polygon, AbstractVector{<: PolyEleme
     )
 end
 
-function plot!(plot::Mesh{<: Tuple{<: AbstractVector{P}}}) where P <: Union{AbstractMesh, Polygon}
+# TODO: for Makie v0.22, GeometryBasics v0.5,
+# switch from AbstractMesh{Polytope{N, T}} to AbstractMesh{N, T}
+function plot!(plot::Mesh{<: Tuple{<: AbstractVector{P}}}) where P <: Union{<: AbstractMesh{<: Polytope{N, T}}, Polygon{N, T}} where {N, T}
```

Comment:
It's just this line that needs to change...AbstractMesh{Polytope{N, T}} becomes AbstractMesh{N, T}